sklearn/utils/estimator_checks.py

```diff
@@ -3105,12 +3105,18 @@ def check_interaction_of_class_and_sample_weight_excluding_class(name, estimator
         estimator_sw.fit(X, y, sample_weight=sample_weight_zero_weight_first)
 
         # Checking if the output is the same for multiple outputs
-        for method in ["predict", "predict_proba", "decision_function", "transform"]:
+        for method in ["predict_proba", "decision_function", "predict"]:
```

Comment:
Why has `tranform` been removed here?

Reply:
I removed it at first because we are dealing only with classifiers, but it has been added back in on the latest commit! :)

By the way, not a problem if you can't review it for now. If you have any other core developers in mind who could take a look, I can work on fixing any issues they identify in the meantime. Alternatively, I can reach out to you again in a couple of months to continue our work from where we left off.